packages/client/src/index.ts

```diff
@@ -38,3 +38,4 @@ export * from './workflow-client';
 export * from './workflow-options';
 export * from './schedule-types';
 export * from './schedule-client';
+export { WorkerBuildIdVersionSets, BuildIdVersionSet, BuildIdOperation } from './build-id-types';
```

Comment:
```suggestion
export * from './task-queue-client';
export { WorkerBuildIdVersionSets, BuildIdVersionSet, BuildIdOperation } from './build-id-types';
```

Reply:
We don't actually want to export everything though